### DISCUSSION TITLE
Set color to specific context for identification.

### DISCUSSION THREAD

**Discussion by murarisumit** [starship/starship#5372] [Q&A]
Is it possible to set some color to specific context. Not sure how to style the string here

`someproject-prod-cluster-eks(blue)`  to blue color prompt
`someproject-stage-cluster-openshift(red)` to red color prompt

```
[kubernetes.context_aliases]
"arn:aws:eks:region:account:cluster/someproject-prod-cluster" = "someproject-prod-cluster-eks"(blue)
"devopshft-cloud-some-stage" = "someproject-stage-cluster-openshift"(red)
"gke-dev-cluster-default-pool-e5f5" = "someproject-dev-cluster-gke"(green)
```

Read https://starship.rs/advanced-config/#style-strings but not able to figure it out

**Comment by murarisumit** [comment]
Seems like duplicate of this discussion: https://github.com/starship/starship/discussions/3273

**Comment by mickimnet** [comment]
There are two open pull request for this feature: #1568 and #614
